Source/WebService/Blackjack.WebService/ClientApp/src/app/components/play-game-page/play-game.component.scss

```diff
@@ -0,0 +1,3 @@
+    public class play_game_page
```

Comment:
This isn't how sass file contents look.

This is still auto generated code. This file should be empty if you have no css to put in it at this time